packages/react/build/mixin.core.js

```diff
@@ -2,24 +2,36 @@ const { Mixin } = require('hops-mixin');
 
 class ReactCoreMixin extends Mixin {
   configureBuild(webpackConfig, { fileLoaderConfig, jsLoaderConfig }, target) {
+    const { experimentalEsbuild } = this.options;
+
     webpackConfig.resolve.extensions.push('.jsx');
     fileLoaderConfig.exclude.push(/\.jsx$/);
     jsLoaderConfig.test.push(/\.jsx$/);
 
-    jsLoaderConfig.options.presets.push(require.resolve('@babel/preset-react'));
-
-    jsLoaderConfig.options.plugins.push(
-      require.resolve('@babel/plugin-transform-flow-strip-types'),
-      require.resolve('@babel/plugin-proposal-class-properties')
-    );
+    if (experimentalEsbuild) {
+      jsLoaderConfig.use[0].options.loader = 'jsx';
```

Comment:
I'm not quite sure, what `use[0]` should be here. I tried logging it with: `DEBUG=hops:webpack:config:develop DEBUG_DEPTH=99` but it seems `use` is not an array?
Maybe it would make sense to make this more robust, like we do in other situations, to find a loader by its name.

Reply:
In the `.experimentalEsbuild` case this is always an array because we have the import component transformation/loader here as well.